bitnami/postgresql-ha/Chart.yaml

```
@@ -27,4 +27,4 @@ name: postgresql-ha
 sources:
   - https://github.com/bitnami/bitnami-docker-postgresql
   - https://www.postgresql.org/
-version: 6.8.3
+version: 6.8.4
```

Please bump the minor version since this is a new feature
```suggestion
version: 6.9.0
```